source/_integrations/vizio.markdown

```diff
@@ -33,9 +36,34 @@ and note its IP address and port number. If you have trouble finding a device yo
 
 ## Pairing
 
-Before adding your device to Home Assistant, you may need to pair it manually. In particular, it is unclear how a sound bar would notify you of a valid auth token. In this case, it might be best to first skip the pairing process entirely, specify a `device_class` of `speaker` in your configuration, and try interacting with the entity to see if you have any success. If the media player controls aren't working, and if specifying different ports as mentioned above doesn't work, you will need to find a way to obtain the auth token during this process.
+This integration requires an access token in order to communicate with TVs (speakers do not need an access token). An access token can be obtained by going through a pairing process, either manually, or through the HA frontend.
+
+### Pair using the HA frontend
+
+ - **Using `configuration.yaml`:** If you have a `vizio` entry in `configuration.yaml` but don't provide an access token value in your configuration, after you initialize HomeAssistant, you will see a Vizio SmartCast device ready to be configured. When you open the configuration modal, you will be guided through the pairing process. While HA will store the access token for the life of your `vizio` entity, it is a good idea to note the access token value displayed in the modal and add it to your `configuration.yaml`. This will ensure that you will not have to go through the pairing process again in the future if you decide to rebuild your HA instance.
+- **Using discovery or manual setup through the Integrations menu:** To initiate the pairing process, simply submit your initial configuration with an empty Access Token value.
+
+### Pair manually using the CLI
+
+The following script, written by [JeffLIrion](https://github.com/JeffLIrion) can be run to obtain an auth token. You will need to replace <IP> with your IP and <PORT> is typically 7345 or 9000.
+
+```bash
+#!bin/bash
```

Comment:
There should be a leading slash. 

```
#!/bin/bash
```

Reply:
Thanks for the correction and thanks for contributing!